tests/consumer_tests/routers/test_inventory.py

```diff
@@ -85,6 +86,25 @@ async def mock_get(path: str, params: str = None):
     assert result["T-456"]["checkout"] and result["T-456"]["checkin"]
 
 
+def test_device_history_with_device_in_use_fail(
```

Comment:
This test is called 'in_use_fail', but can't see how it should in the asserts? Is this expected to fail (`xfail`)? Because it doesn't in the test. Perhaps rewording the name will help.

Reply:
I think I had planned to test it one way but went for the successful route instead, so will rename to `test_device_history_with_device_multiple_checkouts`